pkg/storaged/fsys-tab.jsx

```diff
@@ -118,6 +118,7 @@ export function check_mismounted_fsys(client, path, enter_warning) {
     const [, dir, opts] = get_fstab_config(block);
     const split_options = parse_options(opts);
     const opt_noauto = extract_option(split_options, "noauto");
+    const opt_systemd_automount = extract_option(split_options, "x-systemd.automount");
```

Comment:
This will omit "x-systemd.automount" also from the UI. (It will be removed from the `split_options` array.) We should just check whether it is in it, `split_options.indexOf("x-systemd.automount") != -1`

Reply:
Oh good catch - did not realize it.